### ISSUE TITLE
Cannot run template project

### ISSUE THREAD

**Issue by jared-dykstra** [interbit/interbit#691]
Using:
* node v10.9.0
* npm 6.4.0

Steps:
1. `git clone git@github.com:interbit/interbit.git`
1. `cp -rd interbit/packages/interbit-template/ interbit-template`
1. `cd interbit-template`
1. `npm i`
1. From two different shells, run first:
   1. ` npm run interbit:start`
   1. `npm run start`

This results in a browser tab opening with the following error.  (Note error 2/2 is another instance of the same error that's displayed in error 1/2)

![image](https://user-images.githubusercontent.com/9967320/45388348-f05ba700-b5d5-11e8-9d0e-56965a56fa2a.png)

**Comment by jared-dykstra** [comment]
Possibly change 
```js
payloadFields: Object.keys(action.arguments || action.payload)
```
to 
```js
payloadFields: Object.keys(action.arguments || action.payload || {})
```
?

Although the fact that a `undefined` or `null` action is apparently being dispatched may be a concern?

**Comment by jared-dykstra** [comment]
`v0.4.46` seems to work!  (yay)  The original issue was reported against the default branch (master)

So I'll stick to using the older revision

**Comment by jared-dykstra** [comment]
Separate feature request:  Would you guys be willing to make a repository that only contains `interbit-template` so that the repo could simply be cloned?  It would be a much simpler getting started process.

**Comment by djlmills** [comment]
I agree with your suggestion:
```
payloadFields: Object.keys(action.arguments || action.payload || {})
```

This change was made a few weeks back as it handles the case when an action is dispatched that has a type but not a payload, which is legit in some cases. However, we have not published a new repo version since.

You can work around this by including an empty payload in your action creator.

**Comment by djlmills** [comment]
If you had run the template project from its original location, this would have worked fine. When you run `npm i` it also runs `npm run build:modules` and picks up `interbit-ui-components` locally. When you copied, you are no longer picking up the reference to the local  `interbit-ui-components` and are picking up the published version at `0.4.46` which does not have the fix.

**Comment by nicholeous** [comment]
Closing this issue as it's fixed, requiring publish per #696
